src/Controller/Admin/LoadHelperTrait.php

```diff
@@ -11,7 +11,7 @@ trait LoadHelperTrait {
 	 */
 	protected function loadHelpers(): void {
 		$helpers = [
-			'Time' => ['class' => 'Tools.Time', 'engine' => 'Tools\Utility\FrozenTime'],
+			'Time' => ['class' => 'Tools.Time', 'config' => ['engine' => 'Tools\Utility\FrozenTime']],
```

Comment:
I still dont get how this should not have worked before
```php
        foreach ($objects as $i => $objectName) {
            $config = [];
            if (!is_int($i)) {
                $config = (array)$objectName;
                $objectName = $i;
            }
            [, $name] = pluginSplit($objectName);
            if (isset($config['class'])) {
                $normal[$name] = $config + ['config' => []];
            } else {
                $normal[$name] = ['class' => $objectName, 'config' => $config];
            }
        }
```

Reply:
Sounds like a core bug of sorts, it should merge those into config array IMO
I added a test case that shows that you are right, though: https://github.com/dereuromark/cakephp-queue/commit/a4022b0acefddf6a123f603595a2ed20741a4b9a